Cargo.toml

```diff
@@ -75,13 +75,16 @@ sshkey-attest = { git = "https://github.com/kanidm/webauthn-rs.git", rev = "2218
 # webauthn-rs-proto = { path = "../webauthn-rs/webauthn-rs-proto" }
 # sshkey-attest = { path = "../webauthn-rs/sshkey-attest" }
 
+kanidm-hsm-crypto = { path = "../hsm-crypto" }
```

Comment:
🤔

Reply:
This is because I still need to finish the hsm-crypto side and publish a version.